packages/teams-js/src/public/meeting.ts

```diff
@@ -334,6 +334,47 @@ export namespace meeting {
      * @returns A promise with the updated microphone state
      */
     micMuteStateChangedCallback: (micState: MicState) => Promise<MicState>;
+    /**
+     * Callback for the host to tell the app to change its speaker selection
+     */
+    audioDeviceSelectionChangedCallback?: (selectedDevices: AudioDeviceSelection) => Promise<void>;
+  }
+
+  /**
+   * Interface for AudioDeviceSelection from Teams selection
+   *
+   * @hidden
+   * Hide from docs.
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   *
+   * @beta
+   */
+  export interface AudioDeviceSelection {
+    speaker?: AudioDeviceInfo;
+    microphone?: AudioDeviceInfo;
+    error?: SdkError;
+  }
+
+  /**
+   * Interface for AudioDeviceInfo, includes a device label with the same format as MediaDeviceInfo.label
+   *
+   * Hosted app can use this label to compare it with the device info fetched from `navigator.mediaDevices.enumerateDevices()`
```

Comment:
>`navigator.mediaDevices.enumerateDevices()` [](http://example.com/codeflow?start=83&length=43)

Can you use a `{@link }` tag for this reference?

Reply:
Updated.